ansible/.env.j2

```diff
@@ -0,0 +1,12 @@
+FLASK_APP=todo_app/app
+FLASK_DEBUG=false
+FLASK_ENV=production
+
+SECRET_KEY={{ SECRET_KEY }}
+
+# Trello
+TRELLO_API_KEY={{ TRELLO_API_KEY }}
+TRELLO_API_TOKEN={{ TRELLO_API_TOKEN }}
+TRELLO_BOARD_ID=board-id
+TRELLO_LIST_ID_TODO=todo-list-id
+TRELLO_LIST_ID_DONE=done-list-id
```

Comment:
I typically say the best solution is to extract these as variables like the API key etc. but then set the values as defaults in the playbook, so that you don't need to type (or copy+paste) them all the time, but this works nicely here. I wouldn't bother actually changing this, just one for consideration